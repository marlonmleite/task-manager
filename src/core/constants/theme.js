const theme = {
  colors: {
    primary: '#00b7ce',
  },
}

export default theme
