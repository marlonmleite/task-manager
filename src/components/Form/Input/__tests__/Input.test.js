import React from 'react'
import { shallow } from 'enzyme'
import theme from 'core/constants/theme'
import { InputComponent } from '../'

test('Input', () => {
  const input = shallow(<InputComponent value="abc" theme={theme} />)

  expect(input).toMatchSnapshot()
})

