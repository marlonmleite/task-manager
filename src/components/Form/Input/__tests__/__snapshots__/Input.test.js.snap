// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input 1`] = `
.c0 {
  width: 100%;
  font-size: 15px;
  padding: 5px 5px 5px 0;
  height: 24px;
  border-color: #e5e5e5;
  border-style: solid;
  border-width: 0 0 1px 0;
  border-radius: 0;
  color: #222;
  line-height: 24px;
  background-color: transparent;
  box-sizing: content-box;
}

.c0:focus {
  outline: none;
  border-color: #62C2FA;
}

.c0:disabled {
  cursor: not-allowed;
}

<input
  className="c0"
  type="text"
  value="abc"
/>
`;
